addons/mail/static/src/models/discuss_sidebar_category/discuss_sidebar_category.js

```diff
@@ -239,6 +249,11 @@ registerModel({
             compute: '_computeFilteredCategoryItems',
             readonly: true,
         }),
+        itemNewInputView: one2one('AutocompleteInputView', {
```

Comment:
`InputView` suffix is fine, but "itemNew" is a strange name, already "newItem" would sound better, and probably even "addItem"/"addingItem" according to the name of the other field?